### ISSUE TITLE
It's not working on python3

### ISSUE THREAD

**Issue by eribeiro** [romanlv/trml2pdf#3]
Hi,

I am able to install via pip and have, but whenever I try to import it it gives me the following error:

```
>>> import trml2pdf
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/eribeiro/.virtualenvs/sapl/lib/python3.4/site-packages/trml2pdf/__init__.py", line 18, in <module>
    from trml2pdf import parseString
ImportError: cannot import name 'parseString'
```

Then I changed `__ini__.py`, including a dot as below:

```
from .trml2pdf import parseString
```

Then the error was:

```
>>> import trml2pdf
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/eribeiro/.virtualenvs/sapl/lib/python3.4/site-packages/trml2pdf/trml2pdf.py", line 265
    raise ValueError, "Not enough space"
                    ^
SyntaxError: invalid syntax
```

Looks like the lib is not in sync with the python3 changes, right?

**Comment by willson556** [comment]
Our version has been working on python3 as far as we've tested it. I still need to get the test suite running though. https://github.com/CalPolyResDev/trml2pdf

**Comment by willson556** [comment]
Test suite now runs on Python3 and Python2

**Comment by romanlv** [comment]
fixed with #4

**Comment by eribeiro** [comment]
@willson556 Thanks a LOT! :) Really great job, congrats! :+1:
